packages/databricks-vscode/src/cli/CliWrapper.test.ts

```diff
@@ -5,62 +5,100 @@ import {
     RemoteUri,
     SyncDestinationMapper,
 } from "../sync/SyncDestination";
+import {workspaceConfigs} from "../vscode-objs/WorkspaceConfigs";
 import {promisify} from "node:util";
 import {execFile as execFileCb} from "node:child_process";
 import {withFile} from "tmp-promise";
-import {writeFile} from "node:fs/promises";
-
+import {writeFile, readFile} from "node:fs/promises";
+import {when, spy, reset} from "ts-mockito";
 import {CliWrapper} from "./CliWrapper";
 import path from "node:path";
+import os from "node:os";
+import crypto from "node:crypto";
 import {Context} from "@databricks/databricks-sdk/dist/context";
 import {logging} from "@databricks/databricks-sdk";
 
 const execFile = promisify(execFileCb);
+const cliPath = path.join(__dirname, "../../bin/databricks");
```

Comment:
```suggestion
const cliPath = path.join(__dirname, "..", "..", "bin", "databricks");
```

Reply:
To handle windows.

Reply:
[`path.join` normalizes the path](https://nodejs.org/api/path.html#pathjoinpaths), so there's no need to keep all the parts separate. It doesn't really hurt, but I think it's more readable with the fewer amount of arguments

Reply:
Aah. Nice. Didn't know that. Thanks!